website/src/components/Chat/ChatSection.tsx

```diff
@@ -1,46 +1,38 @@
-import { Card, CardBody, Divider } from "@chakra-ui/react";
-import dynamic from "next/dynamic";
 import { FormProvider, useForm } from "react-hook-form";
 import { ChatConfigFormData } from "src/types/Chat";
 import { getConfigCache } from "src/utils/chat";
+import { useIsomorphicLayoutEffect } from "usehooks-ts";
 
+import { ChatConfigDesktop } from "./ChatConfigDesktop";
 import { ChatConfigSaver } from "./ChatConfigSaver";
 import { useChatContext } from "./ChatContext";
 import { ChatConversation } from "./ChatConversation";
-import { InferencePoweredBy } from "./InferencePoweredBy";
-
-const ChatConfigSummary = dynamic(() => import("./ChatConfigSummary"), { ssr: false });
 
 export const ChatSection = ({ chatId }: { chatId: string | null }) => {
   const { modelInfos } = useChatContext();
 
   console.assert(modelInfos.length > 0, "No model config was found");
 
-  let defaultValues = getConfigCache();
-  if (defaultValues) {
-    const model = modelInfos.find((model) => model.name === defaultValues.model_config_name);
-    if (!model) {
-      defaultValues = null;
-    }
-  }
-  if (!defaultValues) {
-    defaultValues = {
+  const form = useForm<ChatConfigFormData>({
+    defaultValues: {
       ...modelInfos[0].parameter_configs[0].sampling_parameters,
       model_config_name: modelInfos[0].name,
-    };
-  }
+    },
+  });
+
+  useIsomorphicLayoutEffect(() => {
+    const cache = getConfigCache();
+    const model = modelInfos.find((model) => model.name === cache?.model_config_name);
+    if (model && cache) {
+      form.reset(cache);
+    }
+  }, [form.reset, modelInfos]);
```

Comment:
I have to use `useIsomorphicLayoutEffect` here to avoid hydration errors. The UI will flash a bit if they hard refresh the page